test/test_client.py

```diff
@@ -0,0 +1,144 @@
+"""Unit Tests for Wavefront Python SDK.
+
+@author Hao Song (songhao@vmware.com)
+"""
+
+import unittest
+import uuid
+from unittest.mock import ANY
+from unittest.mock import Mock
+from unittest.mock import patch
+
+import requests
+
+from wavefront_sdk.client import WavefrontClient
+from wavefront_sdk.entities.tracing.span_log import SpanLog
+
+
+class TestUtils(unittest.TestCase):
+    """Test direct ingestion."""
+
+    def setUp(self):
+        self._sender = WavefrontClient(
+            'no_server',
+            'no_token',
+            flush_interval_seconds=86400,  # turn off auto flushing
+            enable_internal_metrics=False)
+        self._spans_log_buffer = self._sender._spans_log_buffer
+        self._tracing_spans_buffer = self._sender._tracing_spans_buffer
+        self._response = Mock()
+        self._response.status_code = 200
+
+    def test_send_span_with_span_logs(self):
+
+        # Call code under test
+        self._sender.send_span(
+            'spanName',
+            1635123456789,
+            12345,
+            'localhost',
+            uuid.UUID('11111111-2222-3333-4444-555555555555'),
+            uuid.UUID('11111111-0000-0001-0002-123456789012'),
+            [uuid.UUID('55555555-4444-3333-2222-111111111111')],
+            None,
+            [('application', 'Wavefront'), ('service', 'test-spans')],
+            [SpanLog(
+                 1635123789456000,
+                 {"FooLogKey": "FooLogValue"})])
+
+        # show long diffs
+        self.maxDiff = None
+
+        # Verify span logs correctly emitted
```

Comment:
I agree that I can leave most comments out, but I think this one and **Verify span correctly emitted** comment below do help the reader see what is being tested. For someone totally unfamiliar with the code, it may be hard to understand what the test is doing.